src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/UserManagement/UserManagementController.cs

```diff
@@ -87,7 +87,21 @@ public async Task<ActionResult<Unit>> DeleteUserReleaseRole(Guid id)
                 .RemoveUserReleaseRole(id)
                 .HandleFailuresOrOk();
         }
-
+        
+        /// <summary>
+        ///   Removes all userReleaseRoles and userPublicationRoles for the given user.
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <returns></returns>
+        [HttpDelete("user/{userId:guid}/release-publication-roles/all")]
```

Comment:
I would update this name to `[HttpDelete("user/{userId:guid}/resource-roles/all")]`, as ResourceRole is the common base class of both ReleaseRole and PublicationRole.  This way, this name is a bit more RESTful.

Similarly, `DeleteAllUserReleaseAndUserPublicationRoles` can become `DeleteAllUserResourceRoles`.